content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -912,6 +913,13 @@ users:
       # very large CA data, to this exec plugin as a part of the KUBERNETES_EXEC_INFO
       # environment variable.
       provideClusterInfo: true
+
+      # The contract between the exec plugin and the standard input I/O stream. If the
+      # contract cannot be satisfied, this plugin will not be run and an error will be
+      # returned. Valid values are "Never" (this exec plugin never uses standard input),
+      # "IfAvailable" (this exec plugin wants to use standard input if it is available),
+      # or "Always" (this exec plugin requires standard input to function). Required.
+      interactiveMode: IfAvailable
```

Comment:
I assume people will copy/paste this example blindly, and interactive credential plugins are perhaps not the ideal... would it make sense to make the example be "Never"?

Reply:
ok, i can get on board with `interactive credential plugins are perhaps not the ideal`. maybe s/ideal/the norm/? i originally had `IfAvailable` because it was the default in v1beta1, but i think your argument makes sense. 5c2250779d632c6c71b2a3806eda2f3b89db3e7f